app/views/external_users/claims/_date_attended_fields.html.haml

```diff
@@ -1,23 +1,10 @@
-%fieldset
-  %legend.visuallyhidden
-  .grid-row.extra-data.nested-fields.fee-dates
+.extra-data.nested-fields.fee-dates
+  - if defined?(parent_model_prefix) && defined?(submodel_count)
+    - error_prefix = "#{parent_model_prefix}_date_attended_#{submodel_count}"
+  - else
+    - error_prefix = "date_attended_0"
 
-    - if defined?(parent_model_prefix) && defined?(submodel_count)
-      - error_prefix = "#{parent_model_prefix}_date_attended_#{submodel_count}"
-    - else
-      - error_prefix = "date_attended_0"
+  = link_to_remove_association "Remove", f, {class: "date-action"}
```

Comment:
**[not a blocker]** There's some locales defined for buttons. Would be nice to start moving some of this to it.